code/modules/uplink/uplink_items.dm

```diff
@@ -456,7 +456,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 2
 	include_modes = list(/datum/game_mode/nuclear)
 
-/datum/uplink_item/dangerous/revolver
```

Comment:
you changed the part that doesn't need to be changed hehe

Reply:
wdym? It'd just revert to normal revolver in that state, with no recoil.

Reply:
You need to change line 462 if you want to change the item you get, not this one.

Reply:
this is the datum for the entry in the uplink, it doesn't change the item dispensed (aka the `item` var of this)

Reply:
yeah no im a dummy when i looked at the code the first time i knew to put it there but when i changed this i wasnt paying attention